src/web-mercator-utils.js

```diff
@@ -67,7 +67,7 @@ export function worldToLngLat([x, y], scale) {
 // Returns the zoom level that gives a 1 meter pixel at a certain latitude
 // S=C*cos(y)/2^(z+8)
 export function getMeterZoom({latitude}) {
-  assert(latitude);
+  assert(isFinite(latitude));
```

Comment:
Use `Number.isFinite`

Reply:
Looks like there are more calls to `isFinite` below, we should change all of them to `Number.isFinite`.